src/renderer/util/index.ts

```diff
@@ -1,11 +1,21 @@
 import { constants, promises as fsPromises } from "fs";
-import { resolve } from "path";
+import { readdir, stat } from 'fs/promises';
+import { join, resolve } from "path";
 
 import { trim } from "lodash";
 import { flatten, memoize, uniq } from "lodash";
 
 import { LIST_DELIMITER_SPLIT, MAIN_FONT_WIDTH } from "../constants";
 
+///////////////////////////////////////////////////////////////////
+// TODO - Remove this once multifile support is feature-complete //
+//         Flip the boolean to "true" for testing                //
+///////////////////////////////////////////////////////////////////
+
+const USE_MULTIFILE_ASSUMPTION = false;
```

Comment:
Hmm.  could this read an env variable, so ... I guess ts is interpreted, so it doesn't matter; edit code makes the change happen immediately.

Reply:
Yeah, good point. I'll take a quick peek at what it would take to default to `false` but configure the tests to run such that it's `true`.